sections/04_study.md

```diff
@@ -36,9 +36,16 @@ from gene expression.  Those could be divided into separate sub-sections.*
 
 ### Transcription factors and RNA-binding proteins
 
-*Existing reviews have covered some of these papers rather well and we do not
-want to repeat what has already been well-stated elsewhere.  This could
-be split into two sub-sections or kept very brief.*
+Transcription Factor and RNA-binding proteins are key components for gene regulation, making them very important to understand for higher level biological processes. While high-throughput sequencing techniques such as ChIP-seq have been able to accurately identify binding regions for DNA and RNA proteins, these experiments are both time consuming and expensive. In addition, the sequencing methods do not provide any sort of analysis on the proteins which would lead to a better understanding of the underlying process. Thus, there is a need to computationally predict and understand these binding regions de novo from sequences.
+
+####Transcription Factors
+
+Transcription Factors (TFs) are regulatory proteins which bind to certain locations on a DNA sequence and controls the rate of genetic information transcription. Chromatin immunoprecipitation and massively parallel DNA sequencing (ChIP-seq) technologies are able to identify highly likely binding sites for a certain TF, and databases such as ENCODE have provided ChIP-seq data for hundreds of different TFs across many laboratories. However, ChIP-seq experiments are expensive and time consuming. Since the data that scientists have discovered is available, in silico methods to predict binding sites are of great interest, thus eliminating the need to do new ChIP-seq experiments every time analysis is done on a new sequence. 
+
+In order to computationally predict TFBSs on a DNA sequence, researchers initially used consensus sequences and position weight matrices to match against a test sequence [@tag:Stormo2000_dna]. Simple neural network classifiers were then proposed to differentiate positive and negative binding sites, but did not show significant improvements over the weight matrix matching methods [@tag:Horton1992_assessment]. Later, SVM techniques outperformed the generative methods by using k-mer features \cite{Ghandi2014_enhanced, Setty2015_seqgl}, but string kernel based SVM systems are limited by expensive computational cost proportional to the number of training and testing sequences. Most recently, [@tag:Alipanahi2015_predicting] showed that a convolutional neural network models could achieve state of the art results on the TFBS task, are scalable to a large number of genomic sequences. [@tag:Lanchantin2016_motif] introduced several new convolutional and recurrent neural network models for predicting TFBSs, but it remains unclear which neural architectures work best for all samples and TFs. While neural architectures are rapidly changing and producing better results, it is clear that deep learning can be efficiently and effectively used to do functional prediction on the genome given raw data. 
```

Comment:
the `\cite{}` reference should be changed to the `@tag` system. I like this paragraph here - it nicely (and quickly) establishes a history and sets up the next paragraph for perspective